### ISSUE TITLE
TypeError when Date of Birthday is removed when registering

### ISSUE THREAD

**Issue by saaaaraaaah** [Ding-Lyu/COMPSCI_732_APlus_32#11]
When I am in the process of registering and have entered a date of birth, when I press the x button in the date of birth field, this error comes up due to the field now being empty. 

<img width="897" alt="Screen Shot 2021-06-06 at 7 18 15 PM" src="https://user-images.githubusercontent.com/55167268/120916144-f259e280-c6fb-11eb-9c15-13c1d7b94146.png">

**Comment by Ding-Lyu** [comment]
I think the problem is you pick up the date after 06/06/2021 as I see on the database, pls make sure the DOB is before the 06/06.

Cheers

**Comment by saaaaraaaah** [comment]
> I think the problem is you pick up the date after 06/06/2021 as I see on the database, pls make sure the DOB is before the 06/06. Cheers
> […](#)

Nope, the same error pops up no matter the date. Although, being able to select a date after the current date is an issue too
